saleor/graphql/discount/types/discounts.py

```diff
@@ -51,3 +73,36 @@ class Meta:
     @staticmethod
     def resolve_reason(root: models.OrderDiscount, _info: ResolveInfo):
         return root.reason
+
+    @staticmethod
+    def resolve_total(root: models.OrderLineDiscount, info):
+        return root.amount
+
+
+class OrderLineDiscount(BaseOrderDiscount[models.OrderLineDiscount]):
+    total = graphene.Field(
+        Money, description="The discount amount applied to the line item."
+    )
+    unit = graphene.Field(
```

Comment:
Suggestion: Same with those fields, in `DB`, the `BaseDiscount.amount` is always set, so seems like fields can be `required`